test/commons/dom/get-element-stack.js

```diff
@@ -465,6 +465,27 @@ describe('dom.getElementStack', () => {
       assert.deepEqual(stack, []);
     });
 
+    it('should correctly position children of different stacking contexts', () => {
+      fixture.innerHTML =
+        '<header id="1" style="position: absolute; z-index: 999; height: 50px; width: 100%; top: 0">' +
```

Comment:
It's a bit hard to grok the test without indentation on the `innerHTML`; maybe use the style from `it('should correctly order elements outside of the axe tree', ...)` instead?

Reply:
Good point. I sometimes forget that we can use newer es6 syntax in tests now.